### ISSUE TITLE
Cannot suppress or override icon label in tabs - accessibility issue

### ISSUE THREAD

**Issue by slclark** [ionic-team/ionic#14642]
# Bug Report

**Ionic Info**
Run `ionic info` from a terminal/cmd prompt and paste the output below.

```
cli packages: (/usr/local/lib/node_modules)

    @ionic/cli-utils  : 1.19.2
    ionic (Ionic CLI) : 3.20.0

global packages:

    cordova (Cordova CLI) : 8.0.0 

local packages:

    @ionic/app-scripts : 3.0.1
    Cordova Platforms  : android 7.0.0 ios 4.5.4
    Ionic Framework    : ionic-angular 3.8.0

System:

    Android SDK Tools : 26.1.1
    ios-deploy        : 1.9.2 
    ios-sim           : 6.1.2 
    Node              : v8.9.4
    npm               : 6.0.0 
    OS                : macOS High Sierra
    Xcode             : Xcode 9.4 Build version 9F1027a 

```

**Describe the Bug**
When a tab has an icon and text, both are read out loud when testing with VoiceOver on iOS.  It doesn't seem to be possible to override the output with aria-label or hide the name of the icon from being read out loud.

**Steps to Reproduce**
1. Create an ion-tabs that contains an ion-tab that has both an icon and text 
2. Set [attr.aria-label]="my label" on the ion-tab
2. Enable VoiceOver in iOS.
3. Tap the tab on the device and it reads out the icon name and the text because the aria-label has not been replaced with a custom label.

**Expected Behavior**
I would expect that my custom aria-label would override both the icon and the text.  So instead of saying 'images outline postcards' it would say 'make a custom postcard'

I would also settle for VoiceOver only saying the text of the tab if both text and icons are present.  I'd rather it just say 'postcards' than 'images postcards' or 'images outline postcards'
